test/common/event/scaled_range_timer_manager_test.cc

```diff
@@ -440,45 +532,45 @@ TEST_F(ScaledRangeTimerManagerTest, MultipleTimersWithChangeInScalingFactor) {
 
   manager.setScaleFactor(0.5);
 
-  // Now that the scale factor is 0.5, fire times are 0: T+10, 1: T+13, 2: T+14, 3: T+13.
-  // Advance to timer 2's min.
+  // Now that the scale factor is 0.5, fire times are 0: start 10, 1: start 13, 2: start 14, 3:
```

Comment:
nit: You lost some of the + after the T to start rename.

Reply:
Thanks, fixed.